subprojects/core/src/test/groovy/org/gradle/api/internal/project/NewDefaultProjectTest.groovy

```diff
@@ -33,7 +33,7 @@ class NewDefaultProjectTest extends AbstractProjectBuilderSpec {
 
         then:
         rootTasks.size() == 2
-        rootTasks[project]*.path as Set == [":bar", ":buildEnvironment", ":components", ":dependencies", ":dependencyInsight", ":dependentComponents", ":foo", ":help", ":javaToolchains", ":model", ":outgoingVariants", ":projects", ":properties", ":resolvableConfigurations", ":tasks", ":foo"] as Set
+        rootTasks[project]*.path as Set == [":bar", ":buildEnvironment", ":components", ":dependencies", ":dependencyInsight", ":dependentComponents", ":foo", ":help", ":init", ":javaToolchains", ":model", ":outgoingVariants", ":projects", ":properties", ":resolvableConfigurations", ":tasks", ":wrapper", ":foo"] as Set
```

Comment:
💭 Don't know why init and wrapper weren't showing up here when help and tasks were.  Another case of fixing an unreported issue.

Reply:
The way `init` and `wrapper` tasks are added require certain jars to be on the classpath.  Since you added the `build-init` project, those tasks became available.